src/Interfaces/VTK.jl

```diff
@@ -7,9 +7,34 @@ using Reexport
 using WriteVTK
 
 using ..Meshes
+using ..Nodes
+using ..Elements
 
 @reexport import WriteVTK: vtk_grid
+export write_vtks
 
 # Add code here.
 
+function node_coordinates_matrix(nodes::Vector{N}) where {dim,T,N<:Node{dim,T}}
+    nodes_coords_matrix = Matrix{Float64}(undef, (length(nodes), dim))
+    dict_nodes_enum = Dict{AbstractNode,Int}()
+    for (i, n) in enumerate(nodes)
+        nodes_coords_matrix[i, :] = collect(coordinates(n))
+        dict_nodes_enum[n] = i
```

Comment:
I guess this should be removed from here since these lines are now in https://github.com/ONSAS/ONSAS.jl/pull/417

Reply:
Yep, the Meshes provides those methods.  If u have already implemented it this way, we can refactor the implementation in a different PR.

Reply:
done in [178ce1a](https://github.com/ONSAS/ONSAS.jl/pull/414/commits/178ce1abcfe211d33bb8ce4f89641d2dbc2fa234)